easybuild/easyconfigs/l/LAMMPS/LAMMPS-29Aug2024_install-deepmd-plugin.patch

```diff
@@ -0,0 +1,9 @@
+diff -ruN lammps-stable_29Aug2024.orig/cmake/CMakeLists.txt lammps-stable_29Aug2024/cmake/CMakeLists.txt
```

Comment:
@laraPPr Please add missing patch description (+ author/source), and fix checksum in easyconfig accordingly